### PR TITLE
Put an example to documentation

### PR DESCRIPTION
**Pull request by cagdastulek**
## Proposed changes

There is no example explaining how to use a WebElementJSONObject response as a parameter to `frame` function. I am a newbie, so maybe I am not doing it right but this example code worked for me fine. I am definitely open to suggestions for a better approach.

## Types of changes

What types of changes does your code introduce to WebdriverIO?

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: lib/protocol/frame.js [hunk 1]
```diff
@@ -1,6 +1,12 @@
 /**
  * Change focus to another frame on the page. If the frame id is null,
  * the server should switch to the page's default content.
+ * <example>
+    :Using `element` to find an iframe and providing it to `frame`
```
Comment (christian-bromann) [inline lib/protocol/frame.js]: this line is suppose to be the file name. To stay consistent with the other commands let's change that to be an file name and make the sentence to a comment

**Comment by christian-bromann** [comment]
One comment otherwise LGTM

File: lib/protocol/frame.js (hunk 1 above)
Comment (cagdastulek) [inline lib/protocol/frame.js]: What should be the value of filename? I copied the format from the following link but I didn't understand what the filename stands for (couldn't find it anywhere). See https://github.com/webdriverio/webdriverio/edit/master/lib/commands/waitForExist.js

Comment (christian-bromann) [inline lib/protocol/frame.js]: @cagdastulek just something like `frameExample.js`

**Comment by cagdastulek** [comment]
@christian-bromann Just did the suggested change. Thanks for the help.

File: lib/protocol/frame.js [hunk 2]
```diff
@@ -1,6 +1,13 @@
 /**
  * Change focus to another frame on the page. If the frame id is null,
  * the server should switch to the page's default content.
+ * <example>
+    :frameExample.js
+    # Using `element` to find an iframe and providing it to `frame` method
```
Comment (christian-bromann) [inline lib/protocol/frame.js]: This is not a valid JavaScript comment 😉

Comment (cagdastulek) [inline lib/protocol/frame.js]: Life is hard. Curse of working on multiple languages 😅

**Comment by cagdastulek** [comment]
Done 😄

**Pull request merged**